qa-heal.planx-pla.net/metadata/aggregate_config.json

```diff
@@ -93,7 +93,7 @@
     "adapter_commons": {
         "SAMHDA": {
             "mds_url": "https://qa-heal.planx-pla.net/",
-            "commons_url": "https://qa-heal.planx-pla.net/",
+            "commons_url": "a-heal.planx-pla.net",
```

Comment:
yeah good catch, but I wonder should we just remove this section as a whole since it looks like we don't have SAMHDA metadata records in our MDS anymore